pom.xml

```diff
@@ -58,7 +58,7 @@
     <pdfbox.version>2.0.26</pdfbox.version>
     <keypress.version>2.1.5</keypress.version>
     <jetty.version>9.4.29.v20200521</jetty.version>
-    <netty.version>4.1.77.Final</netty.version>
+    <netty.version>4.1.78.Final</netty.version>
```

Comment:
Is this version upgrade part of the same issue?

Reply:
No, let me merge master branch to not have it listed anymore.